lib/yarv/intern.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module YARV
+  # ### Summary
+  #
+  # `intern` converts top element from stack to a symbol.
+  #
+  # ### TracePoint
+  #
+  # TODO
+  #
+  # ### Usage
+  #
+  # ~~~ruby
+  # :"#{"foo"}"
+  #
+  # == disasm: #<ISeq:<compiled>@<compiled>:1 (1,0)-(1,11)> (catch: FALSE)
```

Comment:
Can you add comments here before the disassembly output? That and the tracepoint (intern doesn't have any tracepoint events, otherwise you'd see something like `[Cc]` after in it in the disassembly).